content/collections/create.en.md

```diff
@@ -157,90 +157,174 @@ For collections to be included in the Open Terms Archive organisation only. For
 
 ## Set up deployment
 
-### On the server
+### Define the inventory
 
-#### Set up a SSH deployment key
+- On your local machine
+- Clone the `<collection_id>-declarations` repository
+- Update following entries in the inventory file `deployment/inventory.yml`:
+  - `<host>` (example: `162.19.74.224`)
+  - `ansible_user: <username>` (example: `debian`)
+  - `ed25519_fingerprint: <server_ssh_fingerprint>` obtained with `ssh-keyscan -t ed25519 <host>` (example: `AAAAC3NzaC1lZDI1ETE5AAAAIJkjE2KIbUcoClK+lKLR5ZvmdXMD/eXWghHdenFeJz4c`)
 
-- Connect to the server with `ssh <username>@<host>` (usual usernames: `debian`, `ubuntu`…)
-- Create a new SSH key: `ssh-keygen -q -N "" -f ~/.ssh/ota-deploy`
+### Add fingerprint to GitHub <collection_name>-declarations settings
+
+- Log in on GitHub using a user account with admin privileges for the `<collection_name>-declarations` repository
+- Go to `https://github.com/OpenTermsArchive/<collection_name>-declarations/settings/secrets/actions`
+- Create the `SERVER_FINGERPRINT` [secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) by using the previously obtained ed25519 fingerprint
+
+### Set up a SSH deployment key
+
+Configure an SSH deployment key to enable the automated deployment process via GitHub actions.
+
+#### Create the SSH key
+
+- Connect to the server: `ssh <username>@<host>`
+- Create a new deployment SSH key: `ssh-keygen -q -N "" -f ~/.ssh/ota-deploy`
 - Add the public key to `authorized_keys`: `cat ~/.ssh/ota-deploy.pub >> ~/.ssh/authorized_keys`
-- Add the private key to the SSH authentication agent: `ssh-add ~/.ssh/ota-deploy` (start the SSH agent before if necessary with `eval ${ssh-agent -s}`)
-- Copy the public and private keys and back them up in the shared passwords repository
+- Add the private key to the SSH authentication agent: `ssh-add ~/.ssh/ota-deploy` (start the SSH agent before if necessary: `eval ${ssh-agent -s}`)
```

Comment:
Why do we need to add the server's private key  to the server's agent? 🤔

Reply:
Great question, it seems completely useless…